packages/vscode-extension/src/webviews/controllers/resultTabs.tsx

```diff
@@ -51,8 +52,8 @@ function renderTable(rows: ResultRows) {
     <table>
       <thead>
         <tr>
-          {Object.keys(rows[0]).map((key) => (
-            <th>{key.toString()}</th>
+          {Object.keys(rows[0]).map((key, i) => (
```

Comment:
I'd like to avoid using the indexes for keys as it's going to create bugs if we'd ever we re-order or remove items from the list. For things that don't have a natural key, I think we should do something like this 

```ts
  function withKey<A>(value: A): { key: string; value: A } {
    return { key: /* id generator like nanoid*/, value };
  }

  const mapItems = useMemo(() => originalItems.map(withKey), [row]);